systemtest/src/main/java/io/strimzi/systemtest/kafkaclients/internalClients/KafkaClients.java

```diff
@@ -86,6 +86,12 @@ public String getConsumerGroup() {
         return consumerGroup;
     }
 
+    public void generateNewConsumerGroup() {
+        final String newConsumerGroup = ClientUtils.generateRandomConsumerGroup();
+        LOGGER.info("Regenerating new consumer group {} for clients {} {}", newConsumerGroup, this.getProducerName(), this.getConsumerGroup());
```

Comment:
```suggestion
        LOGGER.info("Regenerating new consumer group {} for clients {} {}", newConsumerGroup, this.getProducerName(), this.getConsumerName());
```
I assume you mean consumer name instead of group?